clearpath_motor_msgs/msg/LynxSystemProtection.msg

```diff
@@ -1,19 +1,7 @@
 std_msgs/Header header
 
-# Protection states
-uint8 NORMAL=0
-uint8 THROTTLED=1
-uint8 OVERHEATED=2
-uint8 ERROR=3
-
 # System state based on highest protection state of motors
 uint8 system_state
 
-# A300
-uint8 A300_MOTOR_REAR_LEFT=0
-uint8 A300_MOTOR_FRONT_LEFT=1
-uint8 A300_MOTOR_FRONT_RIGHT=2
-uint8 A300_MOTOR_REAR_RIGHT=3
-
 # Individual motor protection states
-uint8[] motor_states
+clearpath_motor_msgs/LynxMotorProtection[] motor_states
```

Comment:
Given recent discussions about trying to avoid changing the messages as much as possible, are we sure we want to make this change here & apply it to Jazzy, or would it be safer/easier to defer this particular change for the next distro and keep Jazzy as-is?

If we hadn't already released support for all the platforms I'd be fine with this, but changing this message definition is going to require rebuilding & updating firmware on every platform, won't it?

It looks like a good change, and it should definitely be included in a future release. But I'm not 100% sure we want to make this change right _now_.

I'm happy to be overruled on this point, I'm just hesitant about message changes if we don't absolutely need them.